src/components/Calculator.js

```diff
@@ -1,31 +1,51 @@
-        <div className="screen dg">0</div>
-        <button className="lightgray" type="button">AC</button>
-        <button className="lightgray" type="button">+/-</button>
-        <button className="lightgray" type="button">%</button>
-        <button className="orange" type="button">+</button>
-        <button className="lightgray" type="button">7</button>
-        <button className="lightgray" type="button">8</button>
-        <button className="lightgray" type="button">9</button>
-        <button className="orange" type="button">x</button>
-        <button className="lightgray" type="button">4</button>
-        <button className="lightgray" type="button">5</button>
-        <button className="lightgray" type="button">6</button>
-        <button className="orange" type="button">-</button>
-        <button className="lightgray" type="button">1</button>
-        <button className="lightgray" type="button">2</button>
-        <button className="lightgray" type="button">3</button>
-        <button className="orange" type="button">/</button>
-        <button className="lightgray" type="button" id="zero">0</button>
-        <button className="lightgray" type="button">.</button>
-        <button className="orange" type="button">=</button>
+        <div className="screen dg">{ next }</div>
+        <button className="lightgray" type="button" onClick={() => this.handleEvaluation('AC')}>AC</button>
+        <button className="lightgray" type="button" onClick={() => this.handleEvaluation('+/-')}>+/-</button>
+        <button className="lightgray" type="button" onClick={() => this.handleEvaluation('%')}>%</button>
+        <button className="orange" type="button" onClick={() => this.handleEvaluation('+')}>+</button>
+        <button className="lightgray" type="button" onClick={() => this.handleEvaluation(7)}>7</button>
+        <button className="lightgray" type="button" onClick={() => this.handleEvaluation(8)}>8</button>
+        <button className="lightgray" type="button" onClick={() => this.handleEvaluation(9)}>9</button>
+        <button className="orange" type="button" onClick={() => this.handleEvaluation('x')}>x</button>
+        <button className="lightgray" type="button" onClick={() => this.handleEvaluation(4)}>4</button>
+        <button className="lightgray" type="button" onClick={() => this.handleEvaluation(5)}>5</button>
+        <button className="lightgray" type="button" onClick={() => this.handleEvaluation(6)}>6</button>
+        <button className="orange" type="button" onClick={() => this.handleEvaluation('-')}>-</button>
+        <button className="lightgray" type="button" onClick={() => this.handleEvaluation(1)}>1</button>
+        <button className="lightgray" type="button" onClick={() => this.handleEvaluation(2)}>2</button>
+        <button className="lightgray" type="button" onClick={() => this.handleEvaluation(3)}>3</button>
+        <button className="orange" type="button" onClick={() => this.handleEvaluation('÷')}>÷</button>
```

Comment:
[OPTIONAL]

- In general, the calculator has an `add` button just above the equal button and is also given in the design. The symbols are interchanged here for `+` and `÷`. I would invite you to change for a better UI experience.

### Your design

![Screenshot from 2021-08-13 10-23-23](https://user-images.githubusercontent.com/22434660/129307533-85daa75e-d500-4151-834e-1042a46fc980.png)

### Given design 

![Screenshot from 2021-08-10 20-48-42](https://user-images.githubusercontent.com/22434660/128893622-b08fc46e-f651-4e05-a86e-7d4bb2c5cefc.png)

Reply:
Well addressed this!